### ISSUE TITLE
Add an installation section

### ISSUE THREAD

**Issue by rexagod** [publiclab/inline-markdown-editor#62]
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update: ** To complete this PR, you need to add an installation section in the `README.md` file that clearly depicts and covers basic steps needed for a first-timer to setup the lib, much of which can be scooped out from the "contributing" section, but should also include any personal steps you think should be mention-worthy which you might have encountered during the process. We expect you to have atleast setup the development environment once on your local machine before attempting this issue.

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

**Comment by skilfullycurled** [comment]
Can I trouble someone to clarify this FTO?  I know you're supposed to only do one FTO but if it's about adding a "how to install the PL site" I am going to be doing this anyway in the very near future.

**Comment by skilfullycurled** [comment]
Oh, I think I see now, do you mean installing of the editor so it can be worked on per the [contribute section](https://github.com/publiclab/inline-markdown-editor/tree/c6342531174ee9dd92538ae63efc1e303cc7e842#contributing) on the README.md?

**Comment by rexagod** [comment]
Exactly. Do you need any help with this, @skilfullycurled?

**Comment by skilfullycurled** [comment]
By exactly, do you mean the installation of the editor or the installation of the entire website?

Which one did I have right?

**Comment by skilfullycurled** [comment]
Ah, I see you updated the FTO.  I need to setup the development environment first.  I've already done an FTO so I don't want to take it, I was just mentioning it because I'll be setting up the other one and in following the instructions already laid out, was going to see if there is anything I could add.  But after that, if no one's claimed it and you'd like me to give a go, I'll come back for sure.  Again, it's really more about the fact that I'll be doing it anyway, but if you want to save it for a true FTO then no problem!

**Comment by rexagod** [comment]
@skilfullycurled, I meant the installation of a demo dev environment. I
guess the best we can do for now, is leave this for someone new, while you
get acquainted with the repository itself. I would however, like to
recommend you a fto-candidate, #71, I guess. Have a look over there, and
let me know what you think!

Thanks!

**Comment by skilfullycurled** [comment]
Thanks @rexagod, that's actually a temping one since I would definitely like to learn a javascript work flow (e.g., grunt) , but given some of the contributions I'd like to be making in the future, I'm trying to work on my Ruby skills, so if any back end work comes your way, definitely let me know!

**Comment by rexagod** [comment]
Sure, no problem, and just so you know, I think @cesswairimu could help you out if you get stuck anywhere in those Ruby issues, but do note that she's quite busy nowadays, so maybe ping her on the high priority ones?

Thanks! It's really nice to see you taking interest across various repositories! :tada:

**Comment by skilfullycurled** [comment]
Yeah, it's nice to see the moving around of the check-in working as designed!

**Comment by unnati914** [comment]
could i work on this?

**Comment by cesswairimu** [comment]
Hi @unnati914 , we would love your help. please go ahead. Thanks!

**Comment by unnati914** [comment]
thank you!
